pyproject.toml

```diff
@@ -152,5 +152,4 @@ reportPrivateUsage = "warning"
 reportUnboundVariable = "warning"
 
 [tool.pytest.ini_options]
-filterwarnings = ['ignore:.*The environment .* is out of date.*']
-# filterwarnings = ['ignore:.*step API.*:DeprecationWarning']
+filterwarnings = ["ignore::DeprecationWarning"]
```

Comment:
Will this filter all warnings given non-gymnasium based deprecation warnings?

Reply:
It does
`filterwarnings = ["ignore::DeprecationWarning:gymnasium.*:"]` should fix it

Also `warnings.filterwarnings("once", "", DeprecationWarning, module=r"^gymnasium\.")` forces DeprecationWarning to be displayed, it solved where it wasn't properly showed to users ([#2685](https://github.com/openai/gym/pull/2685#issue-1166952337),  [#3059](https://github.com/openai/gym/pull/3059#issue-1361065355)

I wonder if it is still needed now